src/frontend/komponenter/Fagsak/TilkjentYtelse/TilkjentYtelseTidslinje.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import '@navikt/helse-frontend-tidslinje/lib/main.css';
+
+import { IPersonBeregning } from '../../../typer/beregning';
+import { useBehandling } from '../../../context/BehandlingContext';
+import { hentAktivVedtakPåBehandlig } from '../../../utils/fagsak';
+import { RessursStatus } from '@navikt/familie-typer';
+import { IPerson } from '../../../typer/person';
+import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import { Tidslinje } from '@navikt/helse-frontend-tidslinje/lib';
+import { formaterPersonIdent, sisteDatoIMnd } from '../../../utils/formatter';
+import { ToggleGruppe } from 'nav-frontend-toggle';
+import FamilieChevron from '../../../ikoner/FamilieChevron';
+import { Knapp } from 'nav-frontend-knapper';
+import TidslinjeEtikett from './TidslinjeEtikett';
+import { NavigeringsRetning, useTidslinje } from '../../../context/TidslinjeContext';
+
+const TilkjentYtelseTidslinje: React.FC = () => {
+    const { åpenBehandling } = useBehandling();
+    const {
+        tidslinjeInput,
+        genererFormatertÅrstall,
+        genererToggleKnapper,
+        naviger,
+        endreSkala,
+        genererRader,
+        aktivEtikett,
+    } = useTidslinje();
+
+    const aktivVedtak =
+        åpenBehandling.status === RessursStatus.SUKSESS
+            ? hentAktivVedtakPåBehandlig(åpenBehandling.data)
+            : undefined;
+
+    const tidslinjeRader = genererRader(aktivVedtak && aktivVedtak.personBeregninger);
+    if (
+        åpenBehandling.status !== RessursStatus.SUKSESS ||
+        !aktivVedtak ||
+        tidslinjeRader.length === 0
+    )
+        return null;
+
+    const personer = åpenBehandling.data.personer;
+
+    return (
+        <>
+            <div className={'tidslinje-header'}>
+                <Undertittel>{genererFormatertÅrstall()}</Undertittel>
+                <div className={'tidslinje-header__controls'}>
+                    <ToggleGruppe
+                        defaultToggles={genererToggleKnapper()}
+                        kompakt
+                        minstEn
+                        onChange={endreSkala}
+                    />
+                    <div className={'tidslinje-header__navigering'}>
+                        <Knapp mini kompakt onClick={() => naviger(NavigeringsRetning.VENSTRE)}>
+                            <FamilieChevron retning={'venstre'} />
+                            <span className="sr-only">Naviger til venstre i tidslinjen</span>
+                        </Knapp>
+                        <Knapp mini kompakt onClick={() => naviger(NavigeringsRetning.HØYRE)}>
+                            <FamilieChevron />
+                            <span className="sr-only">Naviger til høyre i tidslinjen</span>
+                        </Knapp>
+                    </div>
+                </div>
+            </div>
+            <div className={'tidslinje'}>
```

Comment:
Småplukk, men hadde det blitt med riktig om fragment ble til en div med className=tidslinje? Usikker her altså, det bare så litt rart ut :)

Reply:
Hmm, i så fall vil det bli en overflødig div som ikke har noe for seg, og da er det vel best practice med fragment tror jeg? 🤔 Men hva er det som ser rart ut, at den har en tidslinje header, og så selve tidslinjen? I såfall kan jeg f.eks. endre klassenavnet til tidslinjen til f.eks. tidslinje-body? eller noe lignende?

Reply:
Ja, nettopp :) Men ikke noe big deal!